Ball.java

```diff
@@ -0,0 +1,12 @@
+public class Ball {
+int number;
+Colors color;
+public Ball(int number, Colors color){
+   this.number = number;
+   this.color = color;
+    }
+
+    public String toString(){
+    return "Ball" + "color" + number;

```

Comment:
The `toString` method does not correctly concatenate the `color` and `number` fields. It currently returns the string 'Ballcolor' followed by the number. You should use `this.color` and `this.number` to properly include the object's fields in the returned string. Consider using: `return "Ball: " + this.color + " " + this.number;`